Patches/Fell Tribes/ThingDefs_Races/Bear.xml

```diff
@@ -0,0 +1,125 @@
+
+			<li Class="PatchOperationAdd">
+				<!-- === Shouldn't need to change this at all		  === -->
+				<!-- === Unless you changed the name of your BasePawn === -->
+				<!-- === In which case change "BasePawn" to that name === -->
+				<xpath>Defs/ThingDef[@Name="BasePawn"]/comps</xpath>
+				<value>
+					<li>
+						<compClass>CombatExtended.CompPawnGizmo</compClass>
+					</li>
+					<li Class="CombatExtended.CompProperties_Suppressable" />
+				</value>
+			</li>
+
+			<!-- === Patch their default melee attacks === -->
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/AlienRace.ThingDef_AlienRace[defName = "Alien_Bear"]/statBases</xpath>
+				<value>
+					<AimingAccuracy>1</AimingAccuracy>
+					<MeleeDodgeChance>1</MeleeDodgeChance>
+					<MeleeCritChance>1.15</MeleeCritChance>
+					<MeleeParryChance>1</MeleeParryChance>
+					<ReloadSpeed>1</ReloadSpeed>
+					<Suppressability>1</Suppressability>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="Alien_Bear"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>left claws</label>
+							<capacities>
+								<li>Scratch</li>
+							</capacities>
+							<power>7</power>
+							<cooldownTime>1.15</cooldownTime>
+							<linkedBodyPartsGroup>LeftHand</linkedBodyPartsGroup>
+							<armorPenetrationSharp>0.1</armorPenetrationSharp>
+							<armorPenetrationBlunt>0.462</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>right claws</label>
+							<capacities>
+								<li>Scratch</li>
+							</capacities>
+							<power>7</power>
+							<cooldownTime>1.15</cooldownTime>
+							<linkedBodyPartsGroup>RightHand</linkedBodyPartsGroup>
+							<armorPenetrationSharp>0.1</armorPenetrationSharp>
+							<armorPenetrationBlunt>0.462</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>teeth</label>
+							<capacities>
+								<li>Bite</li>
+							</capacities>
+							<power>10</power>
+							<cooldownTime>1.37</cooldownTime>
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+							<armorPenetrationSharp>0.02</armorPenetrationSharp>
+							<armorPenetrationBlunt>0.923</armorPenetrationBlunt>
+							<chanceFactor>0.2</chanceFactor>
+						</li>
+					</tools>
+				</value>
+			</li>
+			<!-- ===================================== -->
+			<!-- === BaseBodySize, ArmorRatings, MoveSpeed, and BaseHealth should already be set. I think. === -->
+			<!-- ===================================== -->
+			<!-- === Adds In Inspector Tabs 	   === -->
+			<!-- === Inventory and all that jazz   === -->
+			<!-- === Again, no need to change 'em  === -->
+			<!-- === Unless you changed it         === -->
+			<!-- === then change 'em			   === -->
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[@Name="BasePawn"]/inspectorTabs/li[.="ITab_Pawn_Gear"]</xpath>

```

Comment:
so to be specific, then, the only parts that really need to be kept is the block from 44 to 96, which handles the default melee attacks & etc.?

Reply:
Correct.

Lines 105 through 118 should be removed in all five of the race files--the races already have the inventory and suppressability gizmos because they inherit it from BasePawn, so it's redundant to try to add it again.